examples/Makefile

```diff
@@ -1,7 +1,22 @@
-all: attosoc.json blinky.json
+DEVICE ?= GW1N-9
+PN ?= GW1NR-LV9QN881C6/I5
+
+all: attosoc.fs blinky.fs
+
+%.fs: %.pnr.json
+	gowin_pack -d ${DEVICE} -o $@ $^
+
+%.pnr.json: %.json
+	nextpnr-gowin --json $^ --write $@ --device ${PN} --cst tec0117.cst
```

Comment:
The `tec0117.cst` still hardcodes it to that particular board

Reply:
Ok, I will make the change.